docs/book/toc.md

```diff
@@ -33,6 +34,7 @@
   * [Hyper-parameter tuning](user-guide/advanced-guide/hyper-parameter-tuning.md)
   * [Scale compute to the cloud](user-guide/advanced-guide/scale-compute-to-the-cloud.md)
   * [Global settings of ZenML](user-guide/advanced-guide/global-settings-of-zenml.md)
+  * [Migrate your old pipelines and steps](user-guide/advanced-guide/migrate-pipelines-steps.md)
```

Comment:
For the sake of consistency (and more direct URLs when working with the docs), I would rename this Markdown file. This might have a cascading effect on the links within the other pages. 
```suggestion
  * [Migrate your old pipelines and steps](user-guide/advanced-guide/migrate-your-old-pipelines-and-steps.md)
```